example/src/screens/Room.tsx

```diff
@@ -32,20 +41,47 @@ export const Room = ({ navigation }: Props) => {
   const participants = Membrane.useRoomParticipants();
   const [focusedParticipantData, setFocusedParticipantData] =
     useState<Participant | null>(null);
+  const { applyPrevOrder, orderAndSave, lru } = ParticipantsTracksWindowManager(
```

Comment:
it looks like `ParticipantsTracksWindowManager` is a hook, in that case it should use `useSomething` convention
also "window manager"? o.o what windows are you managing?

Reply:
Renamed.